release_bot/releasebot.py

```diff
@@ -212,6 +212,12 @@ def release_handler(success):
         return self.new_release
 
     def make_new_pypi_release(self):
+        if not self.new_release.get('pypi'):
+            self.logger.debug('Skipping PyPi release')
+            # return False because if we don't have any pypi release
```

Comment:
but, on the other hand, you can also release to github and consume the release as a github release, not from PyPI

Therefore I think that we should decouple Pypi, Github, Fedora releases completely. Could you please do that change?

Reply:
> but, on the other hand, you can also release to GitHub and consume the release as a GitHub release, not from PyPI

Sorry, I didn't understand. Can you explain it, by giving an example or something?

Reply:
Let's scratch my idea, it could cause more harm than good, let's leave it as it is now.